app/helpers/application_helper.rb

```diff
@@ -26,6 +26,20 @@ def url_for_only_path(args)
     url_for(:only_path => true, **args)
   end
 
+  def api_resource_path(klass)
+    raise "API plugin not loaded" unless defined?(Api::ApiConfig)
+
+    identifier, _ = Api::ApiConfig.collections.find do |_, config|
```

Comment:
Perhaps `Api::ApiConfig` should be cached? Because it looks like it will load `config/api.yml` on every call.

Reply:
Not sure if I can do more caching here ...

Reply:
I thought Api:ApiConfig was built on the config gem and was already cached?